lib/rebaser-server/src/rebase.rs

```diff
@@ -146,7 +146,10 @@ pub async fn perform_rebase(
     }
 
     if updating_head && *workspace.pk() != WorkspacePk::NONE {
-        let all_open_change_sets = ChangeSet::list_open(ctx).await?;
+        //todo(brit): what do we want to do about change sets that haven't
```

Comment:
@fnichol @stack72 - Something I thought about last night. I think it's probably ok for now, but I can imagine folks not wanting this.

Reply:
Yeah that's a good a good question. If we unapprove them, this could happen a *lot* if the workspace is highly active and would receive a lot of incoming operational transforms. Some could affect an approval (changing modelling) while others likely wouldn't (the result of running an action or resource refresh?).